File: content/api/tracing/code_snippets/send_service.rb
require 'net/http'

port = 8126
host = "127.0.0.1"
path = "/v0.3/services"


req = Net::HTTP::Put.new(path, initheader = { 'Content-Type' => 'application/json'})

req.body = {"service_name": {"app": "my-app","app_type": "web"}}

response = Net::HTTP.new(host, port).start {|http| http.request(req) }

File: content/api/tracing/code_snippets/send_service.py
import json
import requests

# Send the service.
headers = {'Content-type': 'application/json'}
data = {"service_name": {"app": "my-app","app_type": "web"}}

requests.put('http://localhost:8126/v0.3/services', headers=headers, data = json.dumps(data))
